examples/react-web/package.json

```diff
@@ -30,7 +30,7 @@
       "@babel/preset-react"
     ],
     "plugins": [
-      "@babel/plugin-proposal-class-properties"
+      ["@babel/plugin-proposal-class-properties", {}, "class-properties"]
```

Comment:
Do we need this in the first place?

Reply:
I dont think so, it was working the same without it. But I didnt want to remove because I dont know what it does

Reply:
let me try to remove and see how it goes

Reply:
I removed it in https://github.com/nicolodavis/boardgame.io/commit/5bf7125d7efc045d4714c1dfb15509809adffae4 and the examples seem to be ok (except Chess), but maybe I overlooked something.

Reply:
yep, you are right, removed it too here